### PR TITLE
Detect Mysql 5.7 and 8.0

### PR DESCRIPTION
**Pull request by andrerom**
NOTE: Should maybe rather be a match for 8.x to account for future 8.*MySQL versions

### CODE REVIEW

File: platformsh-flex-env.php [hunk 1]
```diff
@@ -129,7 +129,7 @@ function doctrineFormatter(array $credentials) : string
             $dbVersion = (false !== $versionPosition) ? substr($type, $versionPosition + 1) : '10.2';
 
             // Doctrine needs the mariadb-prefix if it's an instance of MariaDB server
-            if ($dbVersion !== '5.5') {
+            if ($dbVersion !== '5.5' && $dbVersion !== '5.7' && $dbVersion !== '8.0') {
```
Comment (Crell) [inline platformsh-flex-env.php]:
```suggestion
            if (! in_array($dbVersion, ['5.5', '5.7', '8.0'])) {
```

**Comment by Crell** [comment]
Erf.  I never did like that logic.  I'm not sure what else to do with it, though.  Doctrine does not make this easy.

**Comment by andrerom** [comment]
> Doctrine does not make this easy.

Indeed.. (Also ref #29)

**Comment by Crell** [comment]
I... don't know why tests are not running for this PR.  

Thinking about this further, since we now offer both MySQL and MariaDB, in multiple versions, this whole logic block needs to get more robust.  I'm going to take this branch and noodle with it a bit more, and probably file a new PR with the result.  Please Stand By(tm).

**Comment by Crell** [comment]
Closing in favor of #30.  Please test and review there.

**Pull request closed without merging**